app/src/features/Mappings/CreateMapping.tsx

```diff
@@ -257,6 +221,7 @@ const CreateMapping = (): JSX.Element => {
     activeStep === 3 && handleSubmitCreation();
   };
   const handleCancelClick = () => {
+    resetCreateMapping();
```

Comment:
It would be better to call `resetCreateMapping` when the component unmounts.

If the user goes back via the browser back button then this will not be triggered right ?